src/main/java/wooteco/subway/service/line/LineNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package wooteco.subway.service.line;
+
+public class LineNotFoundException extends RuntimeException {
+	public LineNotFoundException(String message) {
+		super("노선이 존재하지 않습니다. :" + message);
+	}
```

Comment:
```suggestion
	public LineNotFoundException(Long lineId) {
		super("노선이 존재하지 않습니다. :" + lineId);
	}
```
사용처가 모두 line의 id를 받는데 문자열로 바꾸지 말고 Long으로 받아주면 어떨까요?